packages/libpulseaudio/module-sles-sink.c

```diff
@@ -373,7 +339,10 @@ int pa__init(pa_module*m) {
 	//Needed. Don't touch
     ss.channels = 2; 
     ss.format = PA_SAMPLE_S16LE;
-    int forceFormat = atoi(getenv("PROPERTY_OUTPUT_SAMPLE_RATE"));
+	
+    char *forceFormat_env = getenv("PROPERTY_OUTPUT_SAMPLE_RATE");
+    int forceFormat = 0;
+    if (forceFormat_env != NULL && strlen(forceFormat_env) >= 4) forceFormat = atoi(forceFormat_env); //"8000" is 4 symbols
```

Comment:
The `strlen` condition here is pointless. Also I think the line should be splited, no?